app/forms/add_user_email_form.rb

```diff
@@ -15,11 +15,11 @@ def user
     @user ||= User.new
   end
 
-  def submit(user, params)
+  def submit(user, params, request_id = nil)
```

Comment:
As we start adding parameters, I wonder if it might be a good idea to refactor to keyword arguments instead of positional arguments, so that it's clearer from the callsites the purpose of the argument.

```suggestion
  def submit(user:, params:, request_id: nil)
```

Reply:
Another consideration could be to either (a) move some of these values into the constructor or (b) incorporate them into `params`, so that we keep a consistent method signature for form classes as `#submit(params)`.

Related Slack discussion: https://gsa-tts.slack.com/archives/C0NGESUN5/p1701183890866009

Reply:
It works well passing the request_id via parameters. I found a similar pattern here https://github.com/18F/identity-idp/blob/c0dbfeb30934d2c8e5de8617c9f06d4bce6cda3e/app/controllers/sign_up/registrations_controller.rb#L25